osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHitStrong.cs

```diff
@@ -46,18 +44,26 @@ protected override void CheckJudgement(bool userTriggered)
                 Judgement.SecondHit = true;
         }
 
-        protected override bool HandleKeyPress(Key key)
+        public override bool OnReleased(TaikoAction action)
+        {
+            if (action == firstHitAction)
+                firstKeyHeld = false;

```

Comment:
Correct, which will now cause the second strong hit to not be able to be hit perfectly, although there's no reason why this should be the case in the given scenario.